index.d.ts

```diff
@@ -11,7 +29,7 @@ declare module '@testcafe-community/axe' {
       elementRef?: Boolean;
       selectors?: Boolean;
     }
-  ): Promise<AxeResults>;
+  ): Promise<axeCheckI>;
```

Comment:
Actually now that I see this why rename it at all is AxeResults not right anymore?

Reply:
I noticed that _**runAxe**_ returns the object that contains _error_ or _results_: 
```
const runAxe = ClientFunction((context, options = {}) => {
    return new Promise((resolve) => {
        axe.run(context || document, options, (error, results) => {
            resolve({ error, results });
        });
    });
});
```
From my understanding _**axeCheck**_ is also returning this object: 
```
const axeCheck = async (t, context, options={}) => {
    try {
        // skipping the "document-title" rule as there is an issue with testcafe
        // being unable to find the title of a page inside the <head> tag.
        options = options["rules"]['document-title'] = {'enabled': false};
        return await runAxe.with({ boundTestRun: t })(context, options);
    } catch (e) {
        return { error: e };
    }
};
```

If I understand everything correctly the return should be changed from Promise<AxeResults> to Promise<AxeCheck>: 
```
  interface AxeCheck {
      results: AxeResults;
      error?: any;
  }
```

I used `error?: any` because it is unclear which error type aXe could throw, but _results_ are definetely _AxeResults_. 
Please let me know if my logic is incorrect.